app/controllers/steps_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StepsController < ApplicationController
+  include Wicked::Wizard::Translated
+
+  before_action :authenticate_user!
+
+  steps(*User.steps)
+
+  def show
+    render_wizard
```

Comment:
This shows the view template under `app/views/steps` corresponding to the URL of the `GET` request. For example, `GET /steps/3` renders HTML generated from `app/views/steps/stage_3/html.erb`.